easybuild/easyconfigs/m/MySQLdb/MySQLdb-1.2.5-intel-2014b-Python-2.7.8.eb

```diff
@@ -0,0 +1,40 @@
+easyblock = "PythonPackage"
+
+name = 'MySQLdb'
+version = '1.2.5'
+
+homepage = [
+    'http://mysql-python.sourceforge.net/',
+    'https://pypi.python.org/pypi/MySQL-python/',
```

Comment:
ah, wait, this is homepage... just use the PyPI URL here (homepage should be a string, not a list)